### PR TITLE
Notes from WG July 14, 2021

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

File: main/2021/WG-07-14.md [hunk 1]
```diff
@@ -46,4 +46,139 @@ None.
...
+
+Eric Prud’Hommeaux
+
+Conrad Watt
+
+Ryan Hunt
+
+Sergey Rubanov
+
+### Proposals and dicsussions
```
Comment (Ms2ger) [inline main/2021/WG-07-14.md:64]:
```suggestion
### Proposals and discussions
```

File: main/2021/WG-07-14.md [hunk 2]
```diff
@@ -46,4 +46,139 @@ None.
...
+
+DG: any objections then?
+
+<no objections noted>
+
+Consensus. Feature moves to phase 5
+
+#### Publish First Public Working Draft, and auto-publishing
+EP: For our purposes here, “TR” means w3.org TR; it means a doc that conforms to publication rules.
+The previous doc (the rec) is big and unwieldy. I decided to try to write a sphinx to TR tool to convert it automatically: https://github.com/w3c/sphinx-to-tr example output is at http://uu3.org/checkouts/w3c/sphinx-to-tr/out/
```
Comment (ericprud) [inline main/2021/WG-07-14.md:82]: may as well remove the example output line 'cause uu3.org points to my laptop when it's in a DMZ and when i get around to updating the record at gandi.

File: main/2021/WG-07-14.md [hunk 3]
```diff
@@ -46,4 +46,139 @@ None.
...
+RH: I’m not actually a WG member but i’m here to answer questions about FF if anyone has them. We don’t have issues with our implementation, and we’re supportive of moving to phase 5.
+
+DG: any objections then?
+
+<no objections noted>
+
+Consensus. Feature moves to phase 5
+
+#### Publish First Public Working Draft, and auto-publishing
+EP: For our purposes here, “TR” means w3.org TR; it means a doc that conforms to publication rules.
```
Comment (ericprud) [inline main/2021/WG-07-14.md:81]: “TR” means <http://www.w3.org/TR/>;

File: main/2021/WG-07-14.md [hunk 4]
```diff
@@ -46,4 +46,139 @@ None.
...
+It takes a “fake” spec and runs it through ReSpec, which creates a valid TR doc. ReSpec has some material that shows editors, etc. It has a config that has to be edited from time to time. It generates the front matter and table of contents (richer that what sphinx generates)
+
+LW: this looks very nice. The TOC looks actually useful.
+
+EP: the TOC is more detailed, let you refer to numbered sections, etc. It’s scriptable, we can run it as a GH action, and can be integrated into W3C’s auto-publish. So if we get our action set up to output the sphinx, we can hook it up.
+To get this running we have to decide to move to first public working draft (starting over after our REC, basically), and then to decide to do automatic publication.
+
+LW: how does that relate to using an evergreen standard?
+
+EP: technically they are orthogonal. Auto-publication just lets you keep it up to date when you iterate on a working draft. Evergreen is that you can stay in CR, and not have to go to REC every couple years, etc.
```
Comment (ericprud) [inline main/2021/WG-07-14.md:93]: s/and not have to go to REC/and only have to go to REC/

File: main/2021/WG-07-14.md [hunk 5]
```diff
@@ -46,4 +46,139 @@ None.
...
+EP: the TOC is more detailed, let you refer to numbered sections, etc. It’s scriptable, we can run it as a GH action, and can be integrated into W3C’s auto-publish. So if we get our action set up to output the sphinx, we can hook it up.
+To get this running we have to decide to move to first public working draft (starting over after our REC, basically), and then to decide to do automatic publication.
+
+LW: how does that relate to using an evergreen standard?
+
+EP: technically they are orthogonal. Auto-publication just lets you keep it up to date when you iterate on a working draft. Evergreen is that you can stay in CR, and not have to go to REC every couple years, etc. 
+
+LW: we don’t merge stuff too much, it’s mostly like we just did, with a “really done” chunk of work.
+
+EP: yeah when you go to CR you stabilize for 3 months, to update accessibility, internationalization, keep a summary of what’s changed, etc. With evergreen, the patent commitment and review process you did to get to CR, you want to make sure you don’t invalidate the previous patent/accessibility/i18n reviews etc by starting everything over or whatever.
```
Comment (ericprud) [inline main/2021/WG-07-14.md:97]: s/to update accessibility, internationalization/to give accessibility, internationalization etc. a chance to review/

File: main/2021/WG-07-14.md [hunk 6]
```diff
@@ -46,4 +46,139 @@ None.
...
+Mathjax has a way to pickle its output, but there are some issues, that might need patching. Are there currently any scripts that change the content when you load the sphinx HTML output? If so, I haven’t caught them. (i did just include all the scripts that were in the old doc). If there are things we want to freeze that were from scripts that run from sphinx, they'd have to be made to work in JSDOM.
+In Firefox you can switch it to MathML and mathjax just does its thing. Do you know what it does when you do that? Is that output something you’d want to pickle to HTML, e.g. for other browsers? Do you know anything about mathML in the other browsers?
+
+DS: the state of mathML in webkit browsers is... complicated.
+
+EP: the mathml WG seems to be reanimated. It seems like maybe the browsers are back on board. It would be nice to be able to just do that once the support is there.
+
+LW: is it possible to just dynamically generate HTML from the mathML if the browser doesn’t support it?
+
+EP: sort of. It’s what the current version does with mathjax. In the future it might be possible to do it AOT. or we could have both a mathML and mathjax version.
```
Comment (ericprud) [inline main/2021/WG-07-14.md:112]: s/we could have both a mathML and mathjax version/we could have both mathML and mathjax markup/

not what i said, but what i wish i'd said, and i think what folks understood.

File: main/2021/WG-07-14.md [hunk 7]
```diff
@@ -46,4 +46,139 @@ None.
...
+EP: Second proposal: set up autopublish of core spec, JS API, Web API, to be activated at chair's discretion.
+
+DS: Do we have consensus on this, any objections?
+
+<no objections, proposal is accepted>
+
+DG: After merging SIMD, what do I need to do exactly?
+
+EP: The decision means you just let me know when it’s merged and I’ll try to fix up the rest of the script, and get the webmaster to accept the publication request.
+There will be a 2 week wait for the director to [??]
```
Comment (ericprud) [inline main/2021/WG-07-14.md:161]: s/\[\\?\?\]/approve the transition/

(gratuitous escaping is funny, right?)

**Review by ericprud — APPROVED** [review]
fab notes; scribe-for-life!

**Comment by dschuff** [comment]
Thanks! By the way you can suggest edits directly in the GitHub UI... if you know exactly where to look (you'd never be able to discover this IMO). See https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request

**Pull request merged**